src/app/components/articles/article/other/other.component.scss

```diff
@@ -0,0 +1,10 @@
+@use "/src/styles.scss" as *;
+
+.other {
+    margin-right: $big-border;
+}
+
+h2 {
```

Comment:
wrap h2 into other class for better scss

Reply:
Done.